components/Releases/UpdateRelease.jsx

```diff
@@ -394,6 +411,54 @@ export default function UpdateRelease({
             onChange={handleChange}
           />
 
+          {profileData.is_subscribed || profileData.dlcm_friend ? (
+            <>
+              {changeEmbed ? (
+                <>
+                  <div className="input-wrapper">
+                    <label className="label" htmlFor="playerEmbed">
+                      Bandcamp audio player embed
+                    </label>
+                    <PopoverTip
+                      message={`Paste a bandcamp embed iframe here. We set what the player will look like for aesthetic consistency.`}
+                    />
```

Comment:
This is redundant code. Could you move this field into its own component and reuse?